src/app/containers/Caption/index.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { node, string } from 'prop-types';
+import Caption from '../../components/Figure/Caption';
+import Text from '../../components/Text';
+import { ServiceContext } from '../../components/ServiceContext';
+
+const CaptionWithOffscreenText = ({ children }) => (
+  <ServiceContext.Consumer>
+    {({ imageCaptionOffscreenText }) => (
+      <Caption offscreenTextPrefix={imageCaptionOffscreenText}>
```

Comment:
```suggestion
      <Caption>
        <VisuallyHiddenText>{imageCaptionOffscreenText}</VisuallyHiddenText>
```

By doing the above it means the `Caption` component doesn't need to be aware of where the offscreen text is rendered and therefore it is a pure component which renders a `<figcaption>`. This also makes it easier in the future if the offscreen text needs to be a suffix (or both prefix and suffix) depending on the context of the language.